apps/accounts/verify.py

```diff
@@ -0,0 +1,35 @@
+# Don't import any django stuff here.
+
+import requests
+
+
+def fxa_identify(code, config=None):
+    """Get an FxA profile for an access token. If idenficiation fails return
+    an empty dict."""
+    token = get_fxa_token(code, config).get('access_token')
+    if token:
+        return get_fxa_profile(token, config)
+    else:
+        return {}
+
+
+def get_fxa_token(code, config):
+    response = requests.post(config['oauth_uri'] + '/token', data={
+        'code': code,
+        'client_id': config['client_id'],
```

Comment:
alternatively, you could define all functions more like `get_fxa_token(code, config=settings.FXA_CONFIG):`